include/barcodes/qrcode.php

```diff
@@ -1088,7 +1088,7 @@ protected function generateMaskNo($maskNo, $width, $frame) {
 	 protected function makeMaskNo($maskNo, $width, $s, &$d, $maskGenOnly=false) {
 		$b = 0;
 		$bitMask = array();
-		$bitMask = $this->generateMaskNo($maskNo, $width, $s, $d);
+		$bitMask = $this->generateMaskNo($maskNo, $width, $s);
```

Comment:
Note this code has several bugs and it is way behind:
https://github.com/tecnickcom/tc-lib-barcode
It would be better to switch to the new library.

Reply:
The `generateMaskNo` method accepts only 3 params.